xbmc/guilib/FFmpegImage.cpp

```diff
@@ -366,12 +367,61 @@ void CFFmpegImage::FreeIOCtx(AVIOContext** ioctx)
   av_freep(ioctx);
 }
 
+unsigned int CFFmpegImage::GetKDFormat()
```

Comment:
Return type KD_TEX_FMT instead?
unsigned int results in a few static_casts in the PR.